pages/common/matlab.md

```diff
@@ -0,0 +1,16 @@
+# matlab
+
+> Numerical computation environment by MathWorks.
+> More information: <https://uk.mathworks.com/help/matlab/>.
+
+- Run without splash screen during startup:
+
+`matlab -nosplash`
+
+- Execute a MATLAB statement:
+
+`matlab -r {{matlab_statement}}`
```

Comment:
Perhaps quotes would be a good idea here.
```suggestion
`matlab -r "{{matlab_statement}}"`
```
Also, is there a long-form of the `-r` argument here? We generally refer long-form arguments to aid clarity.